script.js

```diff
@@ -161,19 +162,11 @@ function QuickWiki() {
               var dy = d.offset().top;
               var xgap = e.pageX - dx;
               var ygap = e.pageY - dy;
-              if (!$.browser.msie) {
-                e.preventDefault();
```

Comment:
Oh this is for something else entirely, sorry. The default clicking action is already prevented [here][1] and [here][2], so you don't need to take care of that. I wonder if deleting this particular line affects anything at all?

[1]: https://github.com/koustuvsinha/quickwiki/blob/upgrade/script.js#L26
[2]: https://github.com/koustuvsinha/quickwiki/blob/upgrade/script.js#L263